flake.nix

```diff
@@ -124,7 +127,34 @@
                 };
               };
           };
-          formatter = pkgs.alejandra;
+          formatter = let
+            # We need to exclude the plugins/_sources/* files as they are autogenerated
+            # nix formatter only takes a derivation so we need to make a proxy that passes
+            # the correct flags
+            excludeWrapper = {
+              stdenv,
+              alejandra,
+              writeShellScript,
+              ...
+            }:
+              stdenv.mkDerivation {
+                pname = "alejandra-excludes";
+                version = alejandra.version;
+
+                dontUnpack = true;
+                dontBuild = true;
+                installPhase = let
+                  script = writeShellScript "alejandra-excludes.sh" ''
+                    ${alejandra}/bin/alejandra --exclude ./plugins/_sources "$@"
+                  '';
+                in ''
+                  mkdir -p $out/bin
+                  cp ${script} $out/bin/alejandra-excludes
+                  cat $out/bin/alejandra-excludes
```

Comment:
Why do you `cat` the script content here ?

Reply:
Oops forgot to remove after debugging